cspatients/templates/cspatients/patient.html

```diff
@@ -1,137 +1,129 @@
-                          <div>{{ baby.delivery_time|time:"H:i" }}</div>
-                      </li>
-                      <li class="list-group-item py-2">
-                          WEIGHT OF BABY <br/>
-                          <div>{{ baby.baby_weight_grams }}</div>
-                      </li>
-                      <li class="list-group-item py-2">
-                          APGAR 1 <br/>
-                          <div>{{ baby.apgar_1 }}</div>
-                      </li>
-                      <li class="list-group-item py-2">
-                          APGAR 5 <br/>
-                          <div>{{ baby.apgar_5 }}</div>
-                      </li>
-                      <li class="list-group-item py-2">
-                          NICU <br/>
-                          <div>{{ baby.nicu }}</div>
-                      </li>
-                  </ul>
-              {% endfor %}
-          </div>
-      </div>
-      {% else %}
-      <div style="text-align: center;width: 100%;">
-          <p>Sorry. No such patient was found.</p>
+            <li class="list-group-item py-2">
+                <div style="width: 50%;float: left;">
+                    DATE OF DELIVERY <br/>
+                    <div>
+                      {% for baby in patiententry.entry_babies.all|slice:":1" %}
+                        {{ baby.delivery_time|date:"Y-m-d" }}
+                      {% endfor %}
+                    </div>
+                </div>
+                <div style="width: 50%;float: left;">
+                    NUMBER OF BABIES <br/>
+                    <div>{{ patiententry.foetus|default_if_none:"-" }}</div>
+                </div>
+            </li>
+        </ul>
       </div>
-      {% endif %}
   </div>
+
+    <div style="width: 100%; float: left;padding: 20px;">
+        {% for baby in patiententry.entry_babies.all %}
+            <ul class="list-group">
+                <li class="list-group-item py-2">
+                    <h5>Delivery Details: Baby {{ baby.baby_number }}</h5>
+                </li>
+                <li class="list-group-item py-2">
+                    TIME OF DELIVERY <br/>
+                    <div>{{ baby.delivery_time|time:"H:i" }}</div>
+                </li>
+                <li class="list-group-item py-2">
+                    WEIGHT OF BABY <br/>
+                    <div>{{ baby.baby_weight_grams }}</div>
+                </li>
+                <li class="list-group-item py-2">
+                    APGAR 1 <br/>
+                    <div>{{ baby.apgar_1 }}</div>
+                </li>
+                <li class="list-group-item py-2">
+                    APGAR 5 <br/>
+                    <div>{{ baby.apgar_5 }}</div>
+                </li>
+                <li class="list-group-item py-2">
+                    NICU <br/>
+                    <div>{{ baby.nicu }}</div>
+                </li>
+            </ul>
+            {% endfor %}
+        </div>
+    </div>
```

Comment:
This `</div>` or the one right at the bottom needs to be removed for the tests to pass.